docs/run_status.md

```diff
@@ -0,0 +1,124 @@
+# Run Status tracking in Kedro-Viz
+
+The Run Status feature in Kedro-Viz provides tracking and visualisation of your pipeline execution, giving you detailed insights into your workflow's performance and helping you quickly identify and debug issues.
+
+## Overview
+
+Run Status tracks three main aspects of your pipeline execution:
+
+1. **Pipeline execution**: Overall run status, duration, and timing
+2. **Node execution**: Individual node performance, success/failure status, and execution time
+3. **Dataset operations**: Dataset loading and saving operations with size information
```

Comment:
The Workflow view includes:
- Pipeline execution: Overall run status, start/end time, and duration
- Node execution: Success/failure status, execution time, and performance insights
- Dataset operations: Dataset loads and saves, including file sizes